app/components/Views/LockScreen/index.js

```diff
@@ -79,6 +79,17 @@ class LockScreen extends PureComponent {
       Logger.log('Lockscreen::unlockKeychain - authentication successful');
     } catch (error) {
       this.lock();
+
+      if (error?.message) {
+        const errorMessage = error.message;
+
+        // Track vault corruption with enabled state checking
+        trackVaultCorruption(errorMessage, {
```

Comment:
S: no need for the const
```suggestion
        // Track vault corruption with enabled state checking
        trackVaultCorruption(error.message, {
```